ui/app/templates/components/oidc/scope-form.hbs

```diff
@@ -29,6 +29,15 @@
       Scope
     </h1>
   </p.levelLeft>
+  <p.levelRight>
```

Comment:
Design recommended moving the button to the header (before it was an inline link below the `JsonEditor`)

There's a polish task to do the same for the policy form, but the header is in the route file which makes this change outside the scope of this PR - so it's in the Json editor toolbar for now. 

<img width="842" alt="Screenshot 2023-09-27 at 5 52 48 PM" src="https://github.com/hashicorp/vault/assets/68122737/fa86780f-2afe-4318-8714-ef4b1de15cf4">